### PR TITLE
SALTO-4640: Change deploy error messages

### PR DESCRIPTION
**Pull request by shir-reifenberg**
Remove element IDs and improve deploy error messages

---

_Release Notes_: 

_Okta_adapter_:
- Improve deployment error messages

_Jira_adapter_:
- Improve deployment error messages

_Zendesk_adapter_:
- Improve deployment error messages

---
_User Notifications_: 
None

### CODE REVIEW

**Review by alonstern — COMMENTED** [review]
Jira part LGTM 👍

File: packages/zendesk-adapter/src/filters/macro_attachments.ts [hunk 1]
```diff
@@ -313,6 +313,7 @@ const filterCreator: FilterCreator = ({ config, client }) => ({
           errors: [...parentChanges
             .map(getChangeData)
             .map(e => createSaltoElementError({
+              // TODO
```
Comment (edenhassid) [inline packages/zendesk-adapter/src/filters/macro_attachments.ts:316]:
```suggestion

```

File: packages/zendesk-adapter/src/errors.ts [hunk 2]
```diff
@@ -94,12 +94,11 @@ export const getZendeskError = (elemID: ElemID, error: Error): SaltoElementError
       elemID,
     })
   }
-  const baseErrorMessage = `Deployment of ${elemID.typeName} instance ${elemID.name} failed:`
```
Comment (edenhassid) [inline packages/zendesk-adapter/src/errors.ts:97]: maybe we should keep the `deployment falied` part? what do you think?

File: packages/okta-adapter/src/deployment.ts [hunk 3]
```diff
@@ -65,16 +65,16 @@ export const getOktaError = (elemID: ElemID, error: Error): Error => {
     return error
   }
   const { status, data } = error.response
-  const baseErrorMessage = `Deployment of ${elemID.typeName} instance ${elemID.name} failed with status code ${status}:`
```
Comment (edenhassid) [inline packages/okta-adapter/src/deployment.ts:68]: maybe we should keep the `deployment falied` part? what do you think?

**Review by edenhassid — APPROVED** [review]
LGTM! 💯 make sure you remove the TODO before you merge 😄

File: packages/zendesk-adapter/src/errors.ts (hunk 2 above)
Comment (shir-reifenberg) [reply packages/zendesk-adapter/src/errors.ts:97]: Don't think it's necessary cause it's pretty clear from SAAS screen the deployment failed (same in CLI)

Comment (edenhassid) [reply packages/zendesk-adapter/src/errors.ts:97]: ok :)

**Pull request merged**